### ISSUE TITLE
I really want to see how full my extractor is without having to stand next to it

### ISSUE THREAD

**Issue by PlaymintPaul** [playmint/ds#584]
- The model's fill level should reflect the % the extractor is full
- Clicking on it should show the % full bar in the building summary

Related to this: #579

**Comment by chrisfarms** [comment]
the reason we decided to against displaying the plugin-rendered UI for "buildings you are not stading next to" is so that:

* plugin authors are not required add code that validates if there is a unit selected and nearby
* the inevitable large number of plugins that don't bother to do this validation don't make the place look broken

if we accept that those things are not so bad ... then we can enable the building plugins "from a distance", and you'll be able to see the extractor UI and any other building UI 

...and then maybe we find another way to solve the original problems when/if they arrise later.... it was potentially too premature to attempt to solve the "everything might look broken" problem, before we understood it anyway.

tl;dr ... lets just enable plugins from a distance

**Comment by PlaymintPaul** [comment]
But then you get the other problem of things looking usable but your unit isn't close enough...

**Comment by PlaymintPaul** [comment]
We should discuss the tradeoffs and work out if there is a solution that handles both relatively elegantly

**Comment by chrisfarms** [comment]
```
> eat slime
```

**Comment by chrisfarms** [comment]
> work out if there is a solution that handles both relatively elegantly

how about this...

we make it opt-in that your plugin is usable from a distance. 

at the moment your plugin returns a single "default" content block like:

```
[{
   id: 'default',
   html: '<div>my plugin content</div>',
}]
```

maybe you need to return a special content block for remote viewing:

```
[{
   id: 'use',
   html: '<div>I am plugin content being viewed nearby with a unit selected</div>',
},
{
   id: 'view',
   html: '<div>I am plugin content being viewed from afar or without a unit selected</div>'
}]
```


this doesn't _prevent_ the issue of people making broken stuff ... but at least it becomes a conscious choice by the author to implement it, so it's at least likely that the problem gets considered ..... and since it's optional, the default behaviour is just as now (showing nothing from afar if that bit of content returns nothing)

we could then implement this `view` block for the default extractor plugins to show the capacity/stats and get the result you want.

whatcha recon? ... this would be a very small change, so seems like quite good bang for buck and we could easily get it in for Extraction ... con is that it's another thing to document
